features/archive_autocomplete.feature

```diff
@@ -34,3 +34,14 @@ Feature: Archive autocomplete
             {"value": "PUBLISHED-C"}
         ]}
         """
+        When we get "/archive_autocomplete?field=slugline&language=en&resources=archive"
```

Comment:
I'm not sure if we should add that resources param, afaik there should be always suggestions from all configured resources

Reply:
I was just forward thinking there. I will remove it if you don't foresee any use cases

Reply:
It might be useful to limit it, but imo would be also useful to get them all together by default

Reply:
It does that now. If no argument is provided, it uses all registered resources